typescript/src/generator.ts

```diff
@@ -30,15 +30,16 @@ export class Generator {
   response: plugin.google.protobuf.compiler.CodeGeneratorResponse;
 
   constructor() {
-    this.request = plugin.google.protobuf.compiler.CodeGeneratorRequest.create();
-    this.response = plugin.google.protobuf.compiler.CodeGeneratorResponse.create();
+    this.request =
```

Comment:
all these changes are caused by `gts` version change. If you revert to `gts` v1.0.0 and run `npx gts  fix`, these changes will disappear.